HackYourFuture-CPH/Guide-IT
# Endpoint for GET Users

# Description

We want to be able to get users with specific criteria so that we can show those users. I created a GET request for all the users to retrieve a user by id.

Fixes #31 

# How to test?

Once you have run the migration and seeding to your local database, open this branch and run npm run dev. Go to localhost:3000/api/documentation. You will see two GET requests for users. You can run the tests from there, to retrieve all the users and a user by id.

# Checklist
